pkg/sql/txn_state.go

```diff
@@ -145,12 +145,7 @@ func (ts *txnState2) resetForNewSQLTxn(
 	// TODO(andrei): figure out how to close these spans on server shutdown? Ties
 	// into a larger discussion about how to drain SQL and rollback open txns.
 	var sp opentracing.Span
-	var opName string
-	if txnType == implicitTxn {
-		opName = sqlImplicitTxnName
-	} else {
-		opName = sqlTxnName
-	}
+	opName := sqlTxnName
```

Comment:
Is there a reason to keep this variable around?

Reply:
Yes it's used several times below.